src/ui/buttons/styles/button.js

```diff
@@ -126,10 +126,7 @@ export const buttonStyle = `
         margin-top: 10px;
     }
 
-    @media only screen and (max-width: ${ MIN_VAULT_BUTTON_WIDTH }px) {
-        .menu-button {
-            display: none;
-        }
```

Comment:
@ravishekhar @jshawl @gregjopa @wsbrunson, it appears this isn't used. When the menu button isn't visible (when the button container width is <= 249px), it's being pushed out of view by the paypal button (since its width is the same as the button container width; see line 136/133 being applied to the `paypal-button` class below). The menu button is still in the DOM, however, with `display: inline-block`, and it can be right clicked and scrolled into view. Should we go ahead and remove this then?